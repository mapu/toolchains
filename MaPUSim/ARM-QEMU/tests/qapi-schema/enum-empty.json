# An empty enum, although unusual, is currently acceptable
{ 'enum': 'MyEnum', 'data': [ ] }
