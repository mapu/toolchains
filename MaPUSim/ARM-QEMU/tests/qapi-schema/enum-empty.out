[OrderedDict([('enum', 'MyEnum'), ('data', [])])]
[{'enum_name': 'MyEnum', 'enum_values': []}]
[]
